src/taskcollector-TaskCollector.ts

```diff
@@ -208,6 +208,12 @@ export class TaskCollector {
                 mark
             );
         }
+        if (lineText && this.settings.convertEmptyLines) {
+            return this.updateLineText(
+                `- [ ] ${lineText}`,
```

Comment:
Some tests specifically here around the behavior/preservation of leading whitespace would be good (i.e. an indented line should remain indented, even when taskified.. )

Reply:
Not something I originally considered, good catch!